src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -27,9 +28,8 @@
     pass  # just for type annotations
 
 try:
+    from kubernetes import client, watch
     from kubernetes.client.rest import ApiException
-    from kubernetes.client import V1ListMeta, CoreV1Api, V1Pod
```

Comment:
this change makes mypy fail.

Reply:
it works for me ... 
[jolmomar@juanmipc mgr]$ mypy --config-file=../../mypy.ini  rook/module.py  rook/rook_cluster.py
Success: no issues found in 2 source files

Reply:
this import is still missing

Reply:
This import has been replaced by:
`from kubernetes import client, watch`
Do you prefer to have imported explicitly CoreV1Api, V1Pod? ( V1ListMeta only appears in comments)